src/actions/reporting.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import type { Client } from "bugsnag-react-native";
+import type { Dispatch } from "redux";
+import type { State } from "../reducers";
+
+// eslint-disable-next-line import/prefer-default-export
+export const notifyReporter = (error: Error) => (
+  _: Dispatch<>,
+  __: () => State,
+  errorReporter: Client
```

Comment:
I wonder if it would be better to create a top level action, like our INIT action that is just called ERROR with an error property to hold the error data. Then to notify bugsnag, we create a bugsnag middleware that listens for this action. This would mean that we don't need to instantiate the thunk middleware with extra arguments and complicate our action creators.

Reply:
Sure thing, makes sense. Sorry it took me so long to get back to this, I'll fix this up on Saturday